### ISSUE TITLE
Fetch thumbnail for CP

### ISSUE THREAD

**Issue by pl-mnm** [wrav/oembed#15]
Hi, is there a solution to have the embedded media's thumbnail or image available when choosing columns in the CP's "Customize sources" table? Thx in advance.

**Comment by reganlawton** [comment]
Sorry are you about to clarify this request?

**Comment by pl-mnm** [comment]
Sorry about that, I haven't found the exact terms to describe this CP functionality.

In the CP entries section, when you click the little cog in the bottom left corners you can choose the fields that will be visible to the editor:

![Screenshot_2019-07-12 Entries](https://user-images.githubusercontent.com/3905170/61105568-6c4ba780-a4a4-11e9-8451-9986e927fde4.png)

Ideally I'd like to choose oembed's `thumbnail` value to show in the entries list, the same way you can add an image asset's thumbnail. Does that make sense?

**Comment by reganlawton** [comment]
OH!!! You want to see the thumbnail in the element index 🤔This should be doable jsut abit fiddly and I'll have to look at maybe using transforms if the images are too large.

This is totally something I can add, so leave this with me and if I get side tracked please do ping me. I'm just a bit busy with the tail end of a few projects.

**Comment by pl-mnm** [comment]
Thanks @reganlawton . "Element index": these are the words I was looking for!

**Comment by reganlawton** [comment]
This is still on my todo list 👍

**Comment by pl-mnm** [comment]
Hi there, I'm starting a project where I could use this functionality. Is it still on the todo list? Thanks!

**Comment by reganlawton** [comment]
@pl-mnm Yes it is theres a PR (#22) then I just need to review and update now that I've got a heap of other changes out. It's needs a couple of tweaks then it'll be out hoping next day or so.

**Comment by lenvanessen** [comment]
Keep in mind that you'd want to cache the thumbnails, or not do it all. We had issues with embedded assets firing hundreds of requests to get meta information from the admin, breaking the cms.
